views/js/smaily_module.js

```diff
@@ -93,6 +93,34 @@ $(document).ready(function() {
     });
   });
 
+  // Generate RSS product feed URL if options change.
+  $(".smaily-rss-options").change(function(event) {
+    var rss_url_base = smaily_rss_url + '?';
+    var url_parameters = {};
+
+    var rss_limit = $('#SMAILY_RSS_LIMIT').val();
+    if (rss_limit != "") {
+      url_parameters.limit = rss_limit;
+    }
+
+    var rss_sort_by = $('#SMAILY_RSS_SORT_BY').val();
+    if (rss_sort_by != "") {
+      url_parameters.order_by = rss_sort_by;
+    }
+
+    var rss_sort_order = $('#SMAILY_RSS_SORT_ORDER').val();
+    if (rss_sort_order != "") {
+      url_parameters.order_way = rss_sort_order;
+    }
+
+    var rss_category_id = $('#SMAILY_RSS_CATEGORY_ID').val();
+    if (rss_category_id != "all_products") {
+      url_parameters.id_category = rss_category_id;
```

Comment:
Rename `id_category` to `category_id` here.

Reply:
Renamed all to category_id (where possible), it was very annoying keeping track of 2 versions.